helm/examples/metrics/loggingsink.yml

```diff
@@ -0,0 +1,13 @@
+#
+# Defines a logging sink that writes metric reports to the log
```

Comment:
"the log" here seems a little ambiguous (perhaps by design?).
I'm not sure if this adds much, but perhaps something like:
"Defines a logging sink that reports metrics as standard component log entries" ?